frontend-ssr/components/notifications-tooltip/loading-notification/LoadingNotification.module.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../../styles/variables";
```

Comment:
We might not need this file after we extract the single Notification in a separate component - just like we don't have a separate styling for the `ExerciseSubmissionSkeleton.tsx`.